docs/part1-cli.md

```diff
@@ -1,119 +1,260 @@
-# Part 1. Critter classification with bare command-line
+# Part 1: Critter classification with the command-line
 
+## What is the command-line?
 ## What is the command-line?
 
-The command-line is like a magic wand for your computer, allowing you to perform tasks quickly and directly through text commands, much like having a chat with your machine. While it may seem daunting at first glance, its a simple, text-only interface, and incredibly powerful tool that offers a direct way to communicate with and control your computer.
+A command-line interface (CLI) is a means of interacting with a computer program by inputting lines of text called command-lines ([Wikipedia](https://en.wikipedia.org/wiki/Command-line_interface)). While it may seem daunting at first glance, it is a simple, text-only interface, and incredibly powerful tool that offers a direct way to communicate with and control your computer.
 
-In this section of our Nextflow tutorial, we'll explore how to run an existing classification model using OpenAI's CLIP directly from the command-line, to both classify our critters and build a collage of images.
```

Comment:
I actually think it's important to leave in this context that we are using an existing model, and this will be the tool used to perform classification. While we don't need to explain the details of the model, it somewhat answers the "How?".